MAUI/Kanban-Board/Column.md

```diff
@@ -245,20 +245,20 @@ The following output demonstrates the above example code.
 
 {% highlight xaml %}
 
-    <kanban:KanbanColumn x:Name="todoColumn" Title="To Do" MinimumLimit="3" MaximumLimit="5">
-        <kanban:KanbanColumn.ErrorBarSettings>
-            <kanban:KanbanErrorBarSettings Fill="Green" MinValidationFill="Orange" MaxValidationFill="Red" Height="4"/>
-        </kanban:KanbanColumn.ErrorBarSettings>
-    </kanban:KanbanColumn>
+<kanban:KanbanColumn x:Name="todoColumn" Title="To Do" MinimumLimit="3" MaximumLimit="5">
+    <kanban:KanbanColumn.ErrorBarSettings>
+        <kanban:KanbanErrorBarSettings Fill="Green" MinValidationFill="Orange" MaxValidationFill="Red" Height="4"/>
+    </kanban:KanbanColumn.ErrorBarSettings>
+</kanban:KanbanColumn>
 
 {% endhighlight %}
 
 {% highlight C# %}
 
-    todoColumn.ErrorBarSettings.Fill = Colors.Green;
-    todoColumn.ErrorBarSettings.MinValidationFill = Colors.Orange;
-    todoColumn.ErrorBarSettings.MaxValidationFill = Colors.Red;
-    todoColumn.ErrorBarSettings.Height = 4;
+todoColumn.ErrorBarSettings.Fill = Colors.Green;
```

Comment:
@EzhilarasanElangovan31 when compared with XAML, column code is missing. Update it